baldur/README.md

```diff
@@ -0,0 +1,19 @@
+# baldur
+baldur
+
+building
+
+gradle clean build
+docker build -t baldur -f Dockerfile .
+
+checking it
+
+docker run --rm -it baldur:latest /bin/bash
+
+running it on container port 8080
+
+docker run --rm -d -v /:/conf:ro -v /:/logs:rw baldur:latest
```

Comment:
You probably don't mean to suggest mounting / from the host into the container. Something missing from these example commands.

Reply:
Copied from another project, updated to be a more meaningful example.